.gitignore

```diff
@@ -62,3 +62,7 @@ target/
 
 # JANAF cache directory
 thermochem/JANAF_Cache/
+
+# Vim
+*.swp
+tags
```

Comment:
Editor- and OS-specific .gitignore rules should be in a [global gitignore](https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)